Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.32095&attempt_id=544d3f75-0675-41c2-804a-1667989b2800)

<details><summary>Partial log (click to expand)</summary><p>

```
[ 41%] Built target opentx-x9d+-simulator
[ 41%] Linking CXX static library libshared.a
[ 41%] Built target shared
[ 41%] Linking CXX static library libsimulation.a
[ 41%] Built target simulation
[ 41%] Linking CXX static library libqcustomplot.a
[ 41%] Built target qcustomplot
make: *** [Makefile:152: all] Error 2
builder for '/nix/store/6qiv8n9ggjiv0d79ymih3yhl0h6106li-opentx-2.2.1.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/6qiv8n9ggjiv0d79ymih3yhl0h6106li-opentx-2.2.1.drv' failed
```
</p></details>

--- a/pkgs/applications/misc/opentx/default.nix
+++ b/pkgs/applications/misc/opentx/default.nix
@@ -1,0 +1,58 @@
+{ stdenv, fetchFromGitHub
+, cmake, gcc-arm-embedded, python
+, qt5, SDL, gmock
+, dfu-util, avrdude
+}:
+
+let
+
+  version = "2.2.1";
+
+in stdenv.mkDerivation {
+
+  name = "opentx-${version}";
+
+  src = fetchFromGitHub {
+    owner = "opentx";
+    repo = "opentx";
+    rev = version;
+    sha256 = "01lnnkrxach21aivnx1k1iqhih02nixh8c4nk6rpw408p13him9g";
+  };
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = with qt5; [
+    gcc-arm-embedded
+    python python.pkgs.pyqt4
+    qtbase qtmultimedia qttranslations
+    SDL gmock
+  ];
+
+  postPatch = ''
+    sed -i companion/src/burnconfigdialog.cpp -e 's|/usr/.*bin/dfu-util|${dfu-util}/bin/dfu-util|'
+    sed -i companion/src/burnconfigdialog.cpp -e 's|/usr/.*bin/avrdude|${avrdude}/bin/avrdude|'
+  '';
+
+  cmakeFlags = [
+    "-DQT_TRANSLATIONS_DIR=${qt5.qttranslations}/translations"
+    # XXX I would prefer to include these here, though we will need to file a bug upstream to get that changed.
+    #"-DDFU_UTIL_PATH=${dfu-util}/bin/dfu-util"
+    #"-DAVRDUDE_PATH=${avrdude}/bin/avrdude"
+  ];
+
+  meta = with stdenv.lib; {
+    description = "OpenTX Companion transmitter support software";
+    longDescription = ''
+      OpenTX Companion is used for many different tasks like loading OpenTX
+      firmware to the radio, backing up model settings, editing settings and
+      running radio simulators.
+    '';
+    homepage = https://open-tx.org/;
+    license = stdenv.lib.licenses.gpl2;
+    platforms = [ "i686-linux" "x86_64-linux" ];
+    maintainers = with maintainers; [ elitak ];
+  };
+
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -16825,6 +16825,8 @@ with pkgs;
 
   openscad = callPackage ../applications/graphics/openscad {};
 
+  opentx = callPackage ../applications/misc/opentx { };
+
   opera = callPackage ../applications/networking/browsers/opera {};
 
   orca = python3Packages.callPackage ../applications/misc/orca {

